metrics/lib/common.bash

```diff
@@ -95,18 +95,42 @@ check_images()
 # This function performs a docker build on the image names
 # passed in, to ensure that we have the latest changes from
 # the dockerfiles
+build_dockerfile_image()
+{
+	local image="$1"
+	local dockerfile_path="$2"
+
+	echo "docker building $image"
+	if ! docker build --label "$image" --tag "${image}:latest" -< "$dockerfile_path"; then
```

Comment:
I don't think this is reliable - what if I called this function with `image=busybox:latest`? The fix would seem to be to only add on `:latest` if the image doesn't contain a colon. `:latest` is implied I think, but it's good to be explicit here to hint that that is what the function is trying to check.